protocol/protobuf/communities.proto

```diff
@@ -71,6 +71,7 @@ message CommunityTokenPermission {
     CAN_VIEW_CHANNEL = 3;
     CAN_VIEW_AND_POST_CHANNEL = 4;
     BECOME_TOKEN_MASTER = 5;
+    BECOME_TOKEN_OWNER = 6;
```

Comment:
This role has a confusing name. What does it do? As it is worded, it sounds like having this role means you can own any token

Reply:
This is token owner community permission (CommunityTokenPermission_BECOME_TOKEN_OWNER). We need it for displaying owner token community permission